From 0f1a7047b2b5f8ef2ed86f2f5307fd81ad3c343c Mon Sep 17 00:00:00 2001
From: Allen Xu <b45815@freescale.com>
Date: Wed, 1 Oct 2014 17:32:34 -0500
Subject: [PATCH] MLK-9660 ARM: dts reduce QSPI freq on imx6sx sabresd board

According to design spec on QSPI IO freq, reduce the freq to 29MHz.

Non-DQS mode:
-          SDR: 60MHz
-          DDR: 30Mhz

Signed-off-by: Allen Xu <b45815@freescale.com>
(cherry picked from commit b01578a8d466d7420cbc7cfabf984998e8e31657)
---
 arch/arm/boot/dts/imx6sx-sdb.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dts b/arch/arm/boot/dts/imx6sx-sdb.dts
index a41d831..7c2d018 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb.dts
@@ -379,7 +379,7 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 		compatible = "spansion,s25fl128s";
-		spi-max-frequency = <66000000>;
+		spi-max-frequency = <29000000>;
 		spi-nor,ddr-quad-read-dummy = <6>;
 		reg = <0>;
 	};
@@ -388,7 +388,7 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 		compatible = "spansion,s25fl128s";
-		spi-max-frequency = <66000000>;
+		spi-max-frequency = <29000000>;
 		spi-nor,ddr-quad-read-dummy = <6>;
 		reg = <1>;
 	};
-- 
1.8.0

